tests/jvm/src/test/resources/example/Flags_2.13.scala

```diff
@@ -19,8 +19,8 @@ package object p/*<=flags.p.package.*/ {
   final class Y/*<=flags.p.package.Y#*/
   sealed trait Z/*<=flags.p.package.Z#*/
   class AA/*<=flags.p.package.AA#*/(x/*<=flags.p.package.AA#x.*/: Int/*=>scala.Int#*/, val y/*<=flags.p.package.AA#y.*/: Int/*=>scala.Int#*/, var z/*<=flags.p.package.AA#z().*/: Int/*=>scala.Int#*/)
-  class S/*<=flags.p.package.S#*/[@specialized/*=>scala.specialized#*/ T/*<=flags.p.package.S#[T]*/]
-  val List/*=>scala.collection.immutable.List.*/(xs1/*<=flags.p.package.xs1.*/) = ???/*=>scala.Predef.`???`().*/
-  ???/*=>scala.Predef.`???`().*/ match { case List/*=>scala.collection.immutable.List.*/(xs2/*<=local1*/) => ???/*=>scala.Predef.`???`().*/ }
+  class S/*<=flags.p.package.S#*/[@specialized/*=>scala.specialized#*/ /*=>scala.specialized#`<init>`(+2).*/T/*<=flags.p.package.S#[T]*/]
```

Comment:
one of examples where a new marking is added, rather than simply renaming some library class.